tests/e2e/e2e_bank_test.go

```diff
@@ -64,7 +64,11 @@ func (s *IntegrationTestSuite) testBankTokenTransfer() {
 				afterBobUAtomBalance, err = getSpecificBalance(chainEndpoint, bob.String(), uatomDenom)
 				s.Require().NoError(err)
 
-				decremented := beforeAliceUAtomBalance.Sub(tokenAmount).Sub(standardFees).IsEqual(afterAliceUAtomBalance)
+				// check that Alice's balance was reduced by tokenAmount plus at least some amount of standardFees,
+				// since we can't know in advance how much fees will be charged by the feemarket
+				expectedAfterAliceUAtomBalance := beforeAliceUAtomBalance.Sub(tokenAmount).Sub(standardFees)
+				decremented := afterAliceUAtomBalance.Sub(expectedAfterAliceUAtomBalance).IsLT(standardFees)
```

Comment:
the `standardFee` part here is a little bit confusing as the `decrement` is true if any fee was charged, right?
the `standardFee` don't have any impact anymore, so isn't that just `beforeAliceUAtomBalance.Sub(tokenAmount) > afterAliceUAtomBalance`

Reply:
That is correct, but I wanted to emphasize where does the "<" comes from. Since before the feemarket introduction it used to be "==". I can simplify it the way you suggested and just leave the comment above. Would that be ok?

Reply:
Fixed in https://github.com/cosmos/gaia/pull/3028/commits/686b3f0ce7ad7fc5c52b50652fccb7b9e2b2dd25

I did it this way, to still emphasize that the difference must be less than the `standardFees`:

`decremented := beforeAliceUAtomBalance.Sub(tokenAmount).Sub(afterAliceUAtomBalance).IsLT(standardFees)`

Reply:
With the upgrade to a newer version of the feemarket, these assertions are now reverted to what they were before the feemarket was introduced, since Skip changed the way fees are charged by the feemarket module. Now they charge the exact amount that was sent through the tx. They used to charge only the consumed_gas*base_fee + tip, which caused our tests to fail in the first place.

Resolving this conversation as no longer actual.